apps/zbugs/src/pages/issue/issue-composer.tsx

```diff
@@ -82,6 +84,26 @@ export function IssueComposer({isOpen, onDismiss}: Props) {
     }
   };
 
+  const handleImageUpload = (markdown: string) => {
```

Comment:
This code seems to be repeated in issue-composer and issue-detail and comment-composer. Can we factor it into a helper component that is shared?